presets/i18n.js

```diff
@@ -0,0 +1,15 @@
+import { addPlugins } from 'webpack-blocks'
+import { ProvidePlugin } from 'webpack'
+
+export default function(config) {
+  return addPlugins([
+    new ProvidePlugin({
+      gettext: ['ds-frontend/packages/i18n', 'gettext'],
+      pgettext: ['ds-frontend/packages/i18n', 'pgettext'],
+      ngettext: ['ds-frontend/packages/i18n', 'ngettext'],
+      npgettext: ['ds-frontend/packages/i18n', 'npgettext'],
+      gettextNoop: ['ds-frontend/packages/i18n', 'gettextNoop'],
```

Comment:
what the reason for this method? 
`gettext` should return original message when there no translation

Reply:
oh this is based on docs. I've just implement all possible variants from original gettext)

Reply:
https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#using-the-javascript-translation-catalog

Reply:
https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#gettext-noop